packages/cli/src/modules/data-table/data-store.service.ts

```diff
@@ -321,4 +322,25 @@ export class DataStoreService {
 			throw new DataStoreNameConflictError(name);
 		}
 	}
+
+	private validateFilters(dto: ListDataStoreContentQueryDto): void {
+		if (!dto.filter?.filters) {
+			return;
+		}
+
+		for (const filter of dto.filter.filters) {
+			if (['like', 'ilike'].includes(filter.condition)) {
+				if (filter.value === null || filter.value === undefined) {
```

Comment:
do we need this check if we're anyways checking that the typeof `filter.value` has to be string? :D